tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+
+    for quantity in ["density", "bulkVelocity"]:
+        FluidDiagnostics(
+            quantity=quantity,
+            write_timestamps=timestamps,
+        )
+
+    InfoDiagnostics(quantity="particle_count", write_timestamps=timestamps)
+
+
+def check_hier(hier):
+    for ilvl, lvl in hier.levels().items():
```

Ruff

Loop control variable `ilvl` not used within loop body

Rename unused `ilvl` to `_ilvl`

(B007)